### ISSUE TITLE
Keras CNN Accuracy Issue

### ISSUE THREAD

**Issue by vishnu-u** [vishnu-u/Object-detection#1]
I am performing a CNN Classification using MNIST, CIFAR-10, and CIFAR-100 dataset combined. There are a total of 119 classes after removing the common classes among the dataset. The accuracy in training seems to be a good one but the test accuracy is too low. I tried adjusting the learning rate and reconfiguring the model but is not working still. This is the link to my code. 
https://github.com/vishnu-u/Object-detection
 Please suggest a solution.
